release/models/pcep/openconfig-pcep.yang

```diff
@@ -0,0 +1,377 @@
+        key "pce-server-address";
+	
+        description
+          "Configuration and state information for communication with a pce server.";
+
+        leaf pce-server-address {
+          type leafref {
+            path "../config/pce-server-address";
+          }
+          description
+            "IPv4 address of pce server.";
+        }
+
+        container config {
+          description
+            "PCE server configuration.";
+			
+          uses pce-server-config;
+        }
+		
+        container state {
+          config false;
+          description
+            "PCE server state.";
+			
+          uses pce-server-config;
+        }
+		
+	uses pcep-timers-top;
+	uses pcep-authentication-top;
+      }
+    }
+  }
+	
+  grouping pce-server-config {
+    description
+      "Configuration data for establishing a PCEP session with pce servers.";
+
+    leaf pce-server-address {
+      type oc-inet:ip-address;
+      description
+      	"The destination IP address of the PCE server for PCEP service.
+      	Repeat this step to specify multiple PCE servers to provide redundancy.
+      	When multiple candidate servers are specified for a single client,
+      	they are prioritized according to preference.";
+    }
+	
+    leaf pce-server-id {
+      type string;
+      description
+        "A unique name for the pce server.";
+    }
+
+    leaf pce-server-port {
+      type oc-inet:port-number;
+      default 4189;
+      description
+	"The destination TCP port used for PCEP service in the PCE server.";
+    }
+
+    leaf source-address {
+      type oc-inet:ip-address;
+      description
+	"The source IP address used by PCC(Router) to establish PCEP session.";
+    }
+	
+    leaf pce-type {
+      type pce-mode-type;
+      description
+	"The type of PCEP capability supported which is advertised in the Open
+        message when a PCEP session is established.";
+    }
+	
+    leaf preference {
+      type uint8;
+      description
+        "When multiple PCE servers are specified, the candidate PCE server
+        with the highest preference is selected to calculate paths.
+        The greater or lower the value, the higher the preference could
+        dependon vendor implementation. If no preference value is set, indicate
+        the lowest preference or no preference at all.";
+    }
+
+    leaf report-local-lsp {
+      type boolean;
+      description
+      	"Specifies whether the PCC(Router) will advertise LSP existence and state
+      	for LSPs that are not controlled by any PCE (for example,LSPs that are
+      	statically configured at the PCC) but without delegating them.";
+     }
+	
+     leaf pce-initiated-capability-enable {
+       type boolean;
+       default false;
+       description
+      	 "Indicates to PCE that PCC(Router) supports PCE-initiated LSP paths instantiation.
+      	 A PCE can initiate LSPs only for PCCs that advertised this capability.";
+     }
+	
+     leaf sr-capability-enable {
```

Comment:
Actually, more than capabilities, there are three functionalities or features to be enable/disable in a router to talk PCEP with a PCE server:
- Report to PCE local LSPs
- Allow PCE-Initiated tunnels instantiation
- Support for Segment Routing tunnels
Because name autodescribe functionality by itself with two possible states, true or false, with the default option of "false", maybe it's not worth adding a specific container only for this. But, if so I change it.
See ccd5ed7102c5ab780f2ac12b2a2f76bb2ac92220